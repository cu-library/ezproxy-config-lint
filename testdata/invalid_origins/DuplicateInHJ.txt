Title Siku Quanshu
URL http://skqs.yourlib.org
HJ skqs2.yourlib.org
HJ http://skqs2.yourlib.org
DJ skqs.yourlib.org

