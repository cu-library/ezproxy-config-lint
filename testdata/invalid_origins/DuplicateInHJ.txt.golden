testdata/invalid_origins/DuplicateInHJ.txt:4: HJ http://skqs2.yourlib.org ← Origin already seen at "testdata/invalid_origins/DuplicateInHJ.txt:3" (L2005)
